docs/RasterSource.puml

```diff
@@ -0,0 +1,67 @@
+@startuml
```

Comment:
If you're not aware of it, IntelliJ has a nice PlantUML plugin that will render the results while you edit.